File: app/src/main/java/com/mifos/apache/fineract/data/models/loan/CreditWorthinessFactor.java
package com.mifos.apache.fineract.data.models.loan;

/**
 * @author Rajan Maurya
 *         On 12/07/17.
 */

public class CreditWorthinessFactor {

    private String description;
    private Double amount;

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public Double getAmount() {
        return amount;
    }

    public void setAmount(Double amount) {
        this.amount = amount;
    }
}

File: app/src/main/java/org/apache/fineract/data/models/loan/CreditWorthinessFactor.kt
package org.apache.fineract.data.models.loan

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.android.parcel.Parcelize

/**
 * @author Rajan Maurya
 * On 12/07/17.
 */

@Parcelize
data class CreditWorthinessFactor(
    @SerializedName("description") var description: String? = null,
    @SerializedName("amount") var amount: Double? = null
) : Parcelable